src/clusterfuzz/_internal/chrome/crash_uploader.py

```diff
@@ -566,14 +566,12 @@ def get_symbolized_stack_bytes(crash_type, crash_address, symbolized_stack):
     return None
 
 
-def save_crash_info_if_needed(testcase_id, crash_revision, job_type, crash_type,
-                              crash_address, crash_frames):
+def save_crash_info_if_needed(testcase_id, crash_revision, job_type,
```

Comment:
Might be better to shelve changes for crash_uploader for now. I'm going to delete it (today probably)